tutorials/lidar/about-lidar-data.md

```diff
@@ -226,7 +224,7 @@ various methods.
 
 ## Additional Resources
 
-*	What is the [las file format](http://www.asprs.org/committee-general/laser-las-file-format-exchange-activities.html "las file format: ")?
-*	Using .las with Python? [las: python ingest](http://laspy.readthedocs.org/en/latest/tut_background.html)
-*	Specifications for [las v1.3](http://www.asprs.org/a/society/committees/standards/asprs_las_spec_v13.pdf)
+*	What is the <a href="http://www.asprs.org/committee-general/laser-las-file-format-exchange-activities.html" target"_blank">las file format</a>?

```

Comment:
@NEON-Leslie . Hi, this line needs to be updated to  target="_blank" (currently missing the =)